src/table-theme-colors.js

```diff
@@ -1,8 +1,23 @@
-import { isDarkColor } from './table/utils/color-utils';
+import { isDarkColor, isTransparentColor } from './table/utils/color-utils';
 
 export default function tableThemeColors(theme) {
-  const backgroundColor = theme.getStyle('object', 'straightTable', 'backgroundColor');
-  const isBackgroundDarkColor = isDarkColor(backgroundColor);
+  const qvInnerObject = document.querySelector('.qv-object .qv-inner-object');
+  const objectBackgroundColorFromCSS = qvInnerObject && window.getComputedStyle?.(qvInnerObject).backgroundColor;
+  const qvPanelSheet = document.querySelector('.qv-panel-sheet');
```

Comment:
We need to sync this with the sheet background color project so this is the same element they are applying the color too

Reply:
nvm the selector here can easily be changed. In any case, I think we should only look in one place for sheet color and one for object color, so this is good